package.json

```diff
@@ -27,25 +28,25 @@
     },
     "devDependencies": {
         "@babel/core": "^7.22.0",
+        "@babel/parser": "^7.26.7",
         "@babel/preset-env": "^7.22.0",
         "@babel/preset-typescript": "^7.22.0",
+        "@babel/types": "^7.26.7",
         "@biomejs/biome": "^1.9.4",
         "@hotwired/stimulus": "^3.0",
         "@rollup/plugin-typescript": "^12.1.2",
         "@symfony/mock-module": "file:test/fixtures/module",
-        "@symfony/stimulus-testing": "^2.0.0",
```

Comment:
You won't be missed 👋🏻 

But maybe in the future, https://github.com/symfony/stimulus-testing will:
- remove Babel from their dependencies (or put it in peerDeps)
- create subpackages like `@symfony/stimulus-testing-jest` and `@symfony/stimulus-testing-vite`?
- or, being deprecated in favor of documentation...